### DISCUSSION TITLE
CHIME/FRB VOEvent Service Downtime Aug 22, 2022 thru Aug 24, 2022

### DISCUSSION THREAD

**Discussion by emilypetroff** [chime-frb-open-data/community#64] [General]
To all subscribers,

The CHIME/FRB VOEvent Service will be offline from 13:00 UTC on Aug 22 until approximately 11:59 on Aug 24, 2022, to allow for upgrades to be made to the real-time FRB detection pipeline. Stay tuned for updates regarding when the Service will resume.

Best regards,
Emily

**Comment by emilypetroff** [comment]
Upgrade and testing of the real-time FRB detection pipeline has been completed. The service will resume at 14:00 UTC on Thursday 25 August, 2022.

**Comment by antonioaddis** [comment]
Hello Emily,
I am getting issues connecting at chimefrb server, is the service currently online?

Thank you for your assistance!
Best,
Antonio

**Reply by emilypetroff** [reply]
Hi Antonio,
We experienced another issue with connection yesterday and the service was down. I have resumed the service for both emails and xmls so your broker should now be able to connect. Apologies for the inconvenience.
Best,
Emily

**Reply by antonioaddis** [reply]
Thanks Emily, now it works!

Have a nice day!

Best regards,
Antonio
